### DISCUSSION TITLE
FIELD_CHOICES ipam.IPAddress

### DISCUSSION THREAD

**Discussion by ffslcschools** [netbox-community/netbox#13488] [Help Wanted!] [answered]
### NetBox version

3.5.2

### Python version

3.9

### Steps to Reproduce

edit /opt/netbox-3.5.2/netbox/netbox/configuration.py
add additional field choices to FIELD_CHOICES
 'ipam.IPAddress.status+': (
         ('free','Free','teal')
    )
restart netbox.service
New field added shows as the second letter in each word so in the above example I am seeing options for r, r, e. However, if I add a second field both option show correct as DHCPFree and Free.
 'ipam.IPAddress.status+': (
         ('dhcpfree','DHCPFree','orange'),
         ('free','Free','teal')
    )
I do not have this issue in the virtualization.VirtualMachine  name space.

### Expected Behavior

New field choices should show with the proper names out lined in the configuration file.

### Observed Behavior

I have tried changing the words for the new fields and it seems to always be the second letter in each word that gets added as a new field choice. if there is a second field choice added it behaves as expected.

**Comment by jeremystretch** [comment, marked as answer]
There is an error in your Python syntax:

```
'ipam.IPAddress.status+': (
('free','Free','teal')
)
```

You need a comma after the choice to ensure Python processes it as a list item.

**Reply by ffslcschools** [reply]
Even for a single entry? I think the service fails to start if I put a comma in without there being a second entry. I will give it a try and report back.

**Reply by ffslcschools** [reply]
That did allow it to work, so is a comma required after each line regardless of whether it is the last entry or not?

**Reply by markkuleinio** [reply]
A tuple of one item requires the comma, otherwise Python cannot figure out if the parenthesis are meaning a tuple or something else.
